pint_xarray/tests/test_accessors.py

```diff
@@ -222,6 +222,11 @@ def test_error_on_nonsense_units(self, example_unitless_ds):
         with pytest.raises(UndefinedUnitError):
             ds.pint.quantify(units={"users": "aecjhbav"})
 
+    def test_error_indicates_problematic_variable(self, example_unitless_ds):
+        ds = example_unitless_ds
+        with raises_regex(UndefinedUnitError, "users"):
```

Comment:
what's the difference between `raises_regex` and `pytest.raises`' `match` parameter?